lib/services/server/informationReporting.js

```diff
@@ -91,7 +91,7 @@ function ResourceListener(deviceId, objectType, objectId, resourceId, observeStr
             logger.debug(context, 'New data on resource /%s/%s/%s in device [%d]',
                 objectType, objectId, resourceId, deviceId);
 
-            handler(chunk.toString('utf8'));
+            handler(objectType, objectId, resourceId, deviceId, chunk.toString('utf8'));
```

Comment:
This would break code actually using the library, that is expecting the handler to be the first parameter of the handler.